workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.ts

```diff
@@ -89,10 +90,9 @@ export class WBCGroupManagementPlugin implements GroupManagementPlugin {
       await this._userManagementService.addUserToRole(userId, groupId);
```

Comment:
this wasn't checked at either service level or plugin level, you can not add a user to a group that is pending_delete

Reply:
@otlci is handling this is another ticket